bridgestyle/qgis/togeostyler.py

```diff
@@ -609,7 +609,7 @@ def _createSymbolizer(sl, opacity):
         symbolizer = _linePatternFillSymbolizer(sl, opacity)
     elif isinstance(sl, QgsSvgMarkerSymbolLayer):
         symbolizer = _svgMarkerSymbolizer(sl, opacity)
-    elif isinstance(sl, QgsRasterMarkerSymbolLayer):
+    elif type(sl) == "qgis._core.QgsRasterMarkerSymbolLayer":  # is base class with one child (QgsAnimatiedMarkerSymbolLayer) that is not supported
```

Comment:
Thanks for you contribution @benvanbasten-ns.
Do you mean by this that you wish to make sure here the only the `QgsRasterMarkerSymbolLayer` (base) class is processed, but not the `QgsAnimatedMarkerSymbolLayer` class that inherits from it?

In that case, I believe you can do:

```suggestion
    elif type(sl) is QgsRasterMarkerSymbolLayer:  # only support this exact class but no derived classes (e.g. QgsAnimatedMarkerSymbolLayer)
```

I would prefer that instead of a string comparison.